content/en/docs/user-guides/how-to/oci-image-layout.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Manage images as OCI image layout
 description: Build, sign, push, and verify images as OCI image layout
-weight: 10
+weight: 5
 ---
 
 OCI image layout is a directory structure that contains files and folders that refer to an OCI image. OCI image layout is defined in [OCI image spec](https://github.com/opencontainers/image-spec/blob/v1.1.0-rc2/image-layout.md).
```

Comment:
Adding a comment in this line, because I could not find a way to leave comment for the next line which needs changing

~~When you sign an image after it is pushed to a registry, it is possible that you are not signing the correct image. You also may not be certain that the image hasn't been altered.~~ Using OCI image layout, you can sign images on local disk before pushing them to the remote registry. ~~Signing an image before you push it a registry provides helps ensure you are signing the correct image, and consumers of that image can verify that same image.~~

When you sign an image after it is pushed to a registry, it is possible that you are not signing the correct image. You also may not be certain that the image hasn't been altered. Using OCI image layout, you can sign images on local disk before pushing them to the remote registry. Signing an image before you push it a registry provides helps ensure you are signing the correct image, and consumers of that image can verify that same image.